README.md

```diff
@@ -159,6 +159,7 @@ shown below. Can't see yours? Have a look at the [extra platform instructions](h
    ```sh
    curl -fsSL https://starship.rs/install.sh | bash
    ```
+   To update starship itself, rerunnning the script above will replace the current version, keeping configs untouched.
```

Comment:
I'm not really a _linguist_ and hate to nitpick about the wording but this seems more in line with the surrounding text:
```suggestion
   To update the Starship itself, rerun the above script. It will replace the current version without touching Starship's configuration.
```